test/asset-swapper/utils/market_operations_utils/path_test.ts

```diff
@@ -486,21 +484,19 @@ describe('Path', () => {
     });
 });
 
-function createFakeBridgeFill(params: {
-    input: BigNumber;
-    output?: BigNumber;
-    adjustedOutput?: BigNumber;
-    source?: ERC20BridgeSource;
-}): Fill {
-    const { input, output, adjustedOutput, source } = params;
+function createFakeBridgeFill(params: { input: BigNumber; output?: BigNumber; adjustedOutput?: BigNumber }): Fill {
+    const { input, output, adjustedOutput } = params;
     return {
         input,
         output: output || new BigNumber(0),
         adjustedOutput: adjustedOutput || new BigNumber(0),
         gas: 42,
-        source: source || ERC20BridgeSource.UniswapV3,
+        source: ERC20BridgeSource.UniswapV2,
```

Comment:
how come we switched from v3 to v2?

Reply:
good question!

v3 is a special case (😞) that requires a valid filldata (`createFinalBridgeOrderFillDataFromCollapsedFill`) during order creation.

Using uniswap v3 would make `path_test.ts` very cluttered. Since validating uniswap v3 order isn't the focus of `path_test.ts` (it probably belongs to `order.ts`s unit tests) I decided to keep it simple and use v2.